guide/build-config.md

```diff
@@ -234,6 +235,46 @@ You need to specify `group`. Redhat artifacts are then sorted from builds includ
 
 The `MILESTONE` - not implemented yet
 
+The `RESOLVE_ONLY` strategy takes the entry points of the artifacts/extensions of of the product which are supposed to be packaged in repository, resolves the artifact, resolves artifacts managed dependencies. It also takes care of the transitives and package them as well.
+
+ **Necessary configuration**
+ 
+In order to RESOLVE_ONLY strategy to work it requires a link to txt file which contains the artifact or extensions in the format of
+`groupId:artifactId:version:type:classifier` (type and classifier are optional)
+
+*For example*
+
+In build-config.yaml
+
+ ```
+   repositoryGeneration:
+     strategy: RESOLVE_ONLY
+     parameters:
+       extensionsListUrl: "http://link.to.txt.file"
+ ``` 
+ *Text file sample*
+ ```
+ io.quarkus:quarkus-bom-quarkus-platform-properties:1.11.6.Final:properties
+ io.quarkus:quarkus-logging-json-deployment:1.11.6.Final
+ io.quarkus:quarkus-smallrye-opentracing-deployment:1.11.6.Final
+ 
+ ```
+ **Additional Information**
+ Resolve artifacts/extensions in the scope of a bom file
+ In order to resolve the artifacts/extensions in scope of your product bom file, fill `sourceBuild`  and `sourceArtifact` and RESOLVE_ONLY strategy will resolve in the scope of your product bom
+
```

Comment:
What do you mean "in scope" , in context of the BOM and the text configuration file?

Reply:
scope of bom means
when resolving dependencies, resolver migt bring duplicate versions 
but when using scope it will ensure that version of artifact which is getting in repository is of appropriate version according to the given bom